app/src/main/kotlin/com/example/app/api/ChatWebSocketHandler.kt

```
@@ -0,0 +1,88 @@
+                            }
+
+                            val reply =
+                                agent.giveAdvice(
+                                    chatSessionId = chatSessionId,
+                                    input = chatRequest.message,
+                                )
+
+                            val answer = Answer(message = reply, chatSessionId = chatSessionId)
+
+                            objectMapper.writeValueAsString(answer)
+                        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)